### ISSUE TITLE
Other deploy-related actions

### ISSUE THREAD

**Issue by mbrevda** [rollbar/github-deploy-action#5]
What other deploy-related actions are being planned for this action. E.g. uploading source maps?

**Comment by TiagoGouvea** [comment]
Sorry for empty description @mbrevda. I think I found a way to do what I was wanting. 

I think was something related to update the `status` directly from the Action status, but I did it "manually".

This issue can be closed.

**Comment by mrunalk** [comment]
@mbrevda, thanks for your feedback. We have added support for source map uploads in 2.0 release. You can find more details with example on Readme. I will be happy to hear your thoughts on it.

**Comment by mbrevda** [comment]
>  We have added support for source map uploads in 2.0 release

Looking good, would be great to get an example of dynamic uploads (i.e. we don't know the file names in advance due to bundling/code splitting)

**Comment by mrunalk** [comment]
Thanks @mbrevda for your prompt response. Those are all very good suggestions. Consider this release as an initial step in that direction. We will look into how we can dynamically upload source maps. Just curious, do any other tools or actions do dynamic uploads of source maps?

**Comment by mbrevda** [comment]
Yes. [Logrocket's cli](https://docs.logrocket.com/docs/stacktraces#uploading-source-maps-for-a-release) is great example

**Comment by mrunalk** [comment]
@mbrevda, We recently released [rollbar-cli](https://github.com/rollbar/rollbar-cli) with source map upload and deploy functionality. It's still prerelease so your feedback is appreciated.

**Comment by mbrevda** [comment]
Nice! Having a quick glance through the code, am I correct that sourcemaps are uploaded serially (one at a time)? Any reason not to batch these to minimize the amount of time it takes (or even compress all the maps in one archive and upload them in in a single call)?

**Comment by waltjones** [comment]
@mbrevda The Rollbar API cannot handle parallel uploads at this time. Agreed there are some improvements that could be made on the API side that would enable CLI improvements as well.

**Comment by mbrevda** [comment]
In the meantime, you might wish to implement batching client side

**Comment by waltjones** [comment]
Not sure I follow. The API only supports one map file per API request.

**Comment by mbrevda** [comment]
True, but couldn't you make many concurrent requests to the singe-file api?

**Comment by waltjones** [comment]
That may have succeeded for you, but in my testing many of the requests time out and fail. It depends on the size of the files, how many, and other load factors on the API at the specific time of the test.

**Comment by mbrevda** [comment]
Understood. Hope you can update the infrastructure to overcome these things!

**Comment by waltjones** [comment]
Agreed!
